tests/unit/tests_suite_test.go

```diff
@@ -16,10 +16,10 @@ const (
 	CassOperatorChartsPath      = "../../charts/cass-operator"
 	MedusaOperatorChartsPath    = "../../charts/medusa-operator"
 	ReaperOperatorChartsPath    = "../../charts/reaper-operator"
-	DefaultTestNamespace        = "k8ssandra"
+	DefaultTestNamespace        = "k8ssandra-namespace"
```

Comment:
Good idea. The use of "namespace", "release", and "cluster" in the corresponding variables is intended to make the nature of various failures obvious (e.g. when we inadvertently use releaseName instead of clusterName) and make it obvious when we use the wrong one.